streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -446,6 +446,15 @@ void print(final Serde<K> keySerde,
      * @param printed options for printing
      */
     void print(final Printed<K, V> printed);
+    
+    /**
+     * Merges the given streams into one larger stream. 
+     * @param a stream which is to be merged. There is no ordering guarantee between records
```

Comment:
`@param a stream which is to be merged` should go in it's own line at the bottom
Also just realized, that `@return` tag is missing. I would suggest this:
```
/**
 * Merge this stream and the given stream into one larger stream.
 * <p>
 * There is no ordering guarantee between records from this {@code KStream} and records from
 * the provided {@code KStream} in the merged stream.
 * Relative order is preserved within each input stream though (ie, records within one input
 * stream are processed in order).
 *
 * @param a stream which is to be merged into this stream
 * @return a merged stream containing all records from this and the provided {@code KStream}
 */
```